packages/nuka/src/default-controls.tsx

```diff
@@ -268,8 +274,15 @@ export const PagingDots = ({
   );
   const currentSlideBounded = getBoundedIndex(currentSlide, slideCount);
 
+  if (!tabbed) return null;
+
   return (
-    <ul className={pagingDotsContainerClassName} style={listStyles}>
+    <div
+      className={pagingDotsContainerClassName}
+      style={listStyles}
+      aria-label="Choose slide to display."
```

Comment:
Should we let this be customizable with this string as default? (maybe this is more for an eventual port of this for v6 and not this specific use case)